python/python/glide/async_commands/core.py

```diff
@@ -5380,6 +5380,86 @@ async def getex(
             await self._execute_command(RequestType.GetEx, args),
         )
 
+    async def dump(
+        self,
+        key: str,
+    ) -> bytes:
+        """
+        Serialize the value stored at `key` in a Valkey-specific format and return it to the user.
+
+        See https://valkey.io/commands/dump for more details.
+
+        Args:
+            key (str): The `key` of the set.
+
+        Returns:
+            bytes: The serialized value of a set. If `key` does not exist, `None` will be returned.
```

Comment:
```suggestion
            Optional[bytes]: The serialized value of a set. If `key` does not exist, `None` will be returned.
```
Apply on lines 5405 and 5386 too